cicd/gitlab/st/statusTracking.py

```diff
@@ -39,21 +42,22 @@ def parse_result(listOfTasks, checkPublication=False):
                     task['jobsPerStatus']['failed'] -= 1
             total_jobs = sum(task['jobsPerStatus'].values())
             finished_jobs = task['jobsPerStatus']['finished'] if 'finished' in task['jobsPerStatus'] else 0
+            print(finished_jobs)
             published_in_transfersdb = task['publication']['done'] if 'done' in task['publication'] else 0
             failedPublications = task['publication']['failed'] if 'failed' in task['publication'] else 0
             # deal with absurd format (output of a print command !) of outdatasets
             # task['outdatasets'] is of type string and it is like ['datasetname']
             # with string first character being '['
             # beware that soon after submission this is not defined.
             # reference: https://github.com/dmwm/CRABClient/blob/549c4e3b6158e8344315437d1d128f2288551d47/src/python/CRABClient/Commands/status.py#L1059
-            outdataset = eval(task['outdatasets'])[0] if task['outdatasets'] else None
+            outdataset = ast.literal_eval(task['outdatasets'])[0] if task['outdatasets'] else None
```

Comment:
sorry for having misled you !
This scripts always run in python3 in Jenkins, even when we test older CMSSW versions which only have py2.
So let's keep the simpler `eval`.
When we'll add the possibility to test on older CMSSW, we'll need to take care of the correct env. like done now in Jenkins.